### ISSUE TITLE
sporadic timeout in moved-on.chpl under gasnet.numa

### ISSUE THREAD

**Issue by vasslitvinov** [chapel-lang/chapel#13514]
This test:

    test/parallel/taskCount/runningTaskCount/moved-on.chpl

times out sporadically under `gasnet.numa` nightly testing configuration. I can reproduce it as of e9a9d546d5, essentially right after #12922. (I haven't tried c5c0ef94f0 itself, however I do not expect it to be any different w.r.t. this test.)

I reproduced it this way:
* Check out a fresh tree in ptmp on the machine where nightly testing is run.
* Build it under the same environment settings as the nightly testing.
* Run start_test on the test with large -numtrials and stash away the executable and the _real.
* Run the executable repeatedly, concurrently in each of two shells.
* I observed the timeouts after 400 to 1200 trials in one of the shells.

Failure dates observed in nightly testing: 2019-07-11, 2019-07-19, 2019-07-25, 2019-07-26

**Comment by gbtitus** [comment]
@ronawho, might you be interested in taking a look at this, as Qthreads and GASnet person?  During my triage period last week it timed out twice in a row.

**Comment by ronawho** [comment]
Sure

**Comment by gbtitus** [comment]
@mppf, did you tag this as `area: BTR` just because it arose in nightly testing?  You hadn't ruled out an `area: Runtime` root cause, had you?

**Comment by mppf** [comment]
@gbtitus - yes, I didn't find any root cause or anything - I just marked it BTR as a starting point (because issues with no tags... seem to need tagging).

**Comment by daviditen** [comment]
This failed again on 2019-08-02.  I reproduced the failure pretty quickly (under 10 trials twice) with `start_test test/parallel/taskCount/runningTaskCount/moved-on.chpl --num-trials 500`.

**Comment by vasslitvinov** [comment]
Duplicate of https://github.com/Cray/chapel-private/issues/396 .

**Comment by bradcray** [comment]
Just to make sure we're all on the same page, the intention is to track triage-specific sporadic failures and the like on the internal Chapel issues page under the assumption that they're just noise and not value-added for external users and relate more to how we've organized our project at Cray than the state of the software itself.
